rustfmt.toml

```diff
@@ -0,0 +1,4 @@
+comment_width = 80
+wrap_comments = true
```

Comment:
Hm so it's a nightly feature... This produces a lot of not so nice warnings.
But I'm actually more concerned about stability. How stable is this feature, i.e. how much churn does it produce?

Reply:
It seems to be quite stable, the only problem that was reported was when you use markup tables in comments. I guess for now we have to run it manually once in a while to clean up comments.